build.gradle

```diff
@@ -31,10 +31,19 @@ ext.testngVersion = "6.14.3"
 ext.slf4jVersion = "1.7.30"
 ext.javaFtpVersion = "1.6.2"
 ext.fileTransportVersion = "6.0.57"
-ext.commonsVfsVersion="2.6.0"
+ext.commonsVfsVersion="2.2-wso2v1"
```

Comment:
I think we may need to parameterize all the versions here using `gradle.properties`.

Reply:
Parameterized all the versions.